### PR TITLE
[ci skip] Remove reference for `PathSet` from Rails API document

### PR DESCRIPTION
**Pull request by yui-knk**
`ActionView::PathSet` has been NO public api method at least from
Aug 2011 (https://github.com/rails/rails/commit/b14f1c3ad72f7aeef4f725637b835da56bcd7d39),
so remove this from Rails API document.

### CODE REVIEW

**Comment by arthurnn** [comment]
https://github.com/rails/rails/blob/9e25e0e1735f2ccca69679243aa8cf0885104164/actionview/lib/action_view/path_set.rb#L9 indeed thats `nodoc` right now.
@kaspth thoughts?

File: actionview/lib/action_view/view_paths.rb [hunk 1]
```diff
@@ -85,8 +83,8 @@ def view_paths
       # Set the view paths.
       #
       # ==== Parameters
-      # * <tt>paths</tt> - If a PathSet is provided, use that;
-      #   otherwise, process the parameter into a PathSet.
+      # * <tt>paths</tt> - If an Array of String is provided, it gets converted into
```
Comment (kaspth) [inline actionview/lib/action_view/view_paths.rb]: Would amend to `an array of strings` (String should be plural.)

Comment (kaspth) [inline actionview/lib/action_view/view_paths.rb]: Ah, the other documentation has it capitalized so :+1:, but add an s to String still.

File: actionview/lib/action_view/view_paths.rb [hunk 2]
```diff
@@ -85,8 +83,8 @@ def view_paths
...
+      #   the default view paths.
```
Comment (kaspth) [inline actionview/lib/action_view/view_paths.rb]: The other documentation has paths in singular, so let's do that too.

**Comment by kaspth** [comment]
@yui-knk @arthurnn I'm okay with the change, but find the existing documentation pretty inscrutable.

**Comment by rafaelfranca** [comment]
In my opinion that class should be public API. It is private by mistake.

**Comment by schneems** [comment]
It has been :nodoc: since the file's creation https://github.com/rails/rails/commit/73b34e9f75d33dc0709d4ad36c912bdbb8977994#diff-d0d18ae0c1dc3a95b934e325588b6d0e originally committed by @josh

**Comment by rafaelfranca** [comment]
In fact. This whole process need a refactoring. I'll take a closer look.

**Comment by kaspth** [comment]
Yeah, you're right. Having it public makes more sense.

> This whole process need a refactoring.

What do you mean by process here? `PathSet` or that and more parts of Action View?

**Comment by rafaelfranca** [comment]
That and more parts of Action View

**Comment by rafaelfranca** [comment]
TL;DR. You don't need to assign a custom PathSet. Almost everything can be done with strings and `Resolver` instances.

**Comment by kaspth** [comment]
Cool :+1:

@yui-knk Thanks for the pull request, but we're going with @rafaelfranca's suggestion :grin:
